lib/GtkPatch/AboutDialog.vala

```diff
@@ -246,91 +242,90 @@ public class Granite.GtkPatch.AboutDialog : Gtk.Dialog
     /**
      * Creates a new Granite.AboutDialog
      */
-    public AboutDialog()
-    {
+    public AboutDialog () {
         title = "";
         has_resize_grip = false;
         resizable = false;
         deletable = false; // Hide the window's close button when possible
-        set_default_response(ResponseType.CANCEL);
+        set_default_response (Gtk.ResponseType.CANCEL);
 
 
         Granite.Widgets.Utils.set_theming (this, STYLESHEET, null,
                                            Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
         // Set the default containers
-        Box content_area = (Box)get_content_area();
-        Box action_area = (Box)get_action_area();
+        var content_area = (Gtk.Box)get_content_area ();
```

Comment:
If you're already touching the line, you should put a space between the cast and the function :)